docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -963,22 +963,22 @@ The following attributes are used for slider component.
 | `min`              | 0 ~                                      | `elm_slider_min_max_set()`           |
 | `step`             | 0 ~                                      | `elm_slider_step_set()`              |
 | `max`              | 0 ~                                      | `elm_slider_min_max_set()`           |
 | `value`            | 0 ~                                      | `elm_slider_value_set()`             |
-| `inverted`         | `true` or `false` </br>See the [supported properties](#supported_properties) for each profile and version. | `elm_slider_inverted_set()`          |
-| `indicator_format` | `#string` </br>See the [supported properties](#supported_properties) for each profile and version. | `elm_slider_indicator_format_set()`  |
-| `direction`        | `horizontal` or `vertical`See the [supported properties](#supported_properties) for each profile and version. | `elm_slider_horizontal_set()`        |
+| `inverted`         | `true` or `false` </br>See the [supported properties](#properties-supported-for-each-component) for each profile and version. | `elm_slider_inverted_set()`          |
+| `indicator_format` | `#string` </br>See the [supported properties](#properties-supported-for-each-component) for each profile and version. | `elm_slider_indicator_format_set()`  |
+| `direction`        | `horizontal` or `vertical`See the [supported properties](#properties-supported-for-each-component) for each profile and version. | `elm_slider_horizontal_set()`        |
 | `disable`          | `true` or `false`                        | `elm_object_disabled_set()`          |
 | `visible`          | `true` or `false`                        | `evas_object_show()`                 |
-| `style`            | See the [supported properties](#supported_properties) for each profile and version | `elm_object_style_set()`             |
-| `colors`           | `#000000 ~ #ffffff`See the [supported properties](#supported_properties) for each profile and version. | `elm_object_color_class_color_set()` |
+| `style`            | See the [supported properties](#properties-supported-for-each-component) for each profile and version | `elm_object_style_set()`             |
+| `colors`           | `#000000 ~ #ffffff`See the [supported properties](#properties-supported-for-each-component) for each profile and version. | `elm_object_color_class_color_set()` |
 
 ### Spinner (![Spinner](./media/component_attributes_spinner_icon.png))
 
-A [spinner](../../native/guides/ui/efl/component-spinner-m.md) component enables the user to increase or decrease a numeric value by using arrow buttons. **(This component is not supported in wearable circle applications.)**
+A [spinner](../../native/guides/ui/efl/mobile/component-spinner.md) component enables the user to increase or decrease a numeric value by using arrow buttons. **(This component is not supported in wearable circle applications.)**
```

Comment:
It looks `../../../native/guides/ui/efl/mobile/component-spinner.md` is correct.

Reply:
done.